src/index.tsx

```diff
@@ -13,13 +13,13 @@ const root = createRoot(container!)
 root.render(
   <React.StrictMode>
     <TooltipProvider delayDuration={0}>
-      <HashRouter>
+      <BrowserRouter>
         <Routes>
           <Route path="/" element={<Home />} />
           <Route path="/:contract" element={<Contract />} />
           <Route path="/:contract/:method" element={<Contract />} />
         </Routes>
-      </HashRouter>
+      </BrowserRouter>
```

Comment:
Let's do this in a separate branch—it's unrelated to the column width change, and it will require a little extra thought about if and how we handle existing links that we shared to hash-based URLs (I think we included some in our grant application to NEAR Foundation, for example).